RecoJets/JetPlusTracks/python/JetPlusTrackCorrectionsAA_cff.py

```diff
@@ -1,213 +1,66 @@
-
-JPTSisConePu5JetTracksAssociatorAtCaloFace = sisCone5JetTracksAssociatorAtCaloFace.clone()
-JPTSisConePu5JetTracksAssociatorAtCaloFace.jets = cms.InputTag("sisConePu5CaloJets")
-JPTSisConePu5JetTracksAssociatorAtCaloFace.tracks = cms.InputTag("hiGoodMergedTracks")
-
-JPTSisConePu5JetExtender = sisCone5JetExtender.clone()
-JPTSisConePu5JetExtender.jets = cms.InputTag("sisConePu5CaloJets")
-JPTSisConePu5JetExtender.jet2TracksAtCALO = cms.InputTag("JPTSisConePu5JetTracksAssociatorAtCaloFace")
-JPTSisConePu5JetExtender.jet2TracksAtVX = cms.InputTag("JPTSisConePu5JetTracksAssociatorAtVertex")
-
-# Anti-Kt
-from RecoJets.JetAssociationProducers.ak5JTA_cff import*
-
-JPTAntiKtPu5JetTracksAssociatorAtVertex = ak5JetTracksAssociatorAtVertex.clone()
-JPTAntiKtPu5JetTracksAssociatorAtVertex.jets = cms.InputTag("akPu5CaloJets")
-JPTAntiKtPu5JetTracksAssociatorAtVertex.tracks = cms.InputTag("hiGoodMergedTracks")
-
-JPTAntiKtPu5JetTracksAssociatorAtCaloFace = ak5JetTracksAssociatorAtCaloFace.clone()
-JPTAntiKtPu5JetTracksAssociatorAtCaloFace.jets = cms.InputTag("akPu5CaloJets")
-JPTAntiKtPu5JetTracksAssociatorAtCaloFace.tracks = cms.InputTag("hiGoodMergedTracks")
-
-JPTAntiKtPu5JetExtender = ak5JetExtender.clone()
-JPTAntiKtPu5JetExtender.jets = cms.InputTag("akPu5CaloJets")
-JPTAntiKtPu5JetExtender.jet2TracksAtCALO = cms.InputTag("JPTAntiKtPu5JetTracksAssociatorAtCaloFace")
-JPTAntiKtPu5JetExtender.jet2TracksAtVX = cms.InputTag("JPTAntiKtPu5JetTracksAssociatorAtVertex")
-
-### ---------- Task, Sequences
-
-# IC5
-
-JPTrecoJetAssociationsIconePu5Task = cms.Task(
-    trackExtrapolator,
-    JPTiterativeConePu5JetTracksAssociatorAtVertex,
-    JPTiterativeConePu5JetTracksAssociatorAtCaloFace,
-    JPTiterativeConePu5JetExtender
-    )
-JPTrecoJetAssociationsIconePu5 = cms.Sequence(JPTrecoJetAssociationsIconePu5Task)
-
-JetPlusTrackCorrectionsIconePu5Task = cms.Task(
-    JPTrecoJetAssociationsIconePu5,
-    JetPlusTrackZSPCorJetIconePu5
-    )
-JetPlusTrackCorrectionsIconePu5 = cms.Sequence(JetPlusTrackCorrectionsIconePu5Task)
-
-# SC5
-
-JPTrecoJetAssociationsSisConePu5Task = cms.Task(
-    trackExtrapolator,
-    JPTSisConePu5JetTracksAssociatorAtVertex,
-    JPTSisConePu5JetTracksAssociatorAtCaloFace,
-    JPTSisConePu5JetExtender
-    )
-JPTrecoJetAssociationsSisConePu5 = cms.Sequence(JPTrecoJetAssociationsSisConePu5Task)
+JetPlusTrackZSPCorJetAntiKtPu4.JetTracksAssociationAtVertex = cms.InputTag("JPTAntiKtPu4JetTracksAssociatorAtVertex")
+JetPlusTrackZSPCorJetAntiKtPu4.JetTracksAssociationAtCaloFace = cms.InputTag("JPTAntiKtPu4JetTracksAssociatorAtCaloFace")
+JetPlusTrackZSPCorJetAntiKtPu4.JetSplitMerge = cms.int32(2)
+JetPlusTrackZSPCorJetAntiKtPu4.UseTrackQuality = cms.bool(False)
+JetPlusTrackZSPCorJetAntiKtPu4.UseMuons = cms.bool(False)
+JetPlusTrackZSPCorJetAntiKtPu4.UseElectron = cms.bool(False)
+JetPlusTrackZSPCorJetAntiKtPu4.EfficiencyMap = cms.string("CondFormats/JetMETObjects/data/CMSSW_538HI_TrackNonEff.txt")
```

Comment:
as in RecoJets/JetPlusTracks/python/PATJetPlusTrackCorrections_cff.py
```suggestion
JetPlusTrackZSPCorJetAntiKtPu4.JetTracksAssociationAtVertex = "JPTAntiKtPu4JetTracksAssociatorAtVertex"
JetPlusTrackZSPCorJetAntiKtPu4.JetTracksAssociationAtCaloFace = "JPTAntiKtPu4JetTracksAssociatorAtCaloFace"
JetPlusTrackZSPCorJetAntiKtPu4.JetSplitMerge = 2
JetPlusTrackZSPCorJetAntiKtPu4.UseTrackQuality = False
JetPlusTrackZSPCorJetAntiKtPu4.UseMuons = False
JetPlusTrackZSPCorJetAntiKtPu4.UseElectron = False
JetPlusTrackZSPCorJetAntiKtPu4.EfficiencyMap = "CondFormats/JetMETObjects/data/CMSSW_538HI_TrackNonEff.txt"
```

Reply:
Hi Slava, I corrected this item. As for example: I put it to
/afs/cern.ch/work/k/kodolova/public/HMUMUBB/CMSSW_11_2_X_2020-07-28-2300/src/WORK/JetShapeJPTAnalysis_mc_s_sql_qcd_pat2.py
I do not know where to put it either.